### ISSUE TITLE
[Bug] [Mysql-CDC] Mysql-CDC to Doris 提示 No suitable driver

### ISSUE THREAD

**Issue by luojinjiang2** [apache/seatunnel#5106]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

使用Mysql-CDC同步数据到Doris数据库，提交任务报错，提示No suitable driver
1.有按照官方文档安装好SeaTunnel 2.3.2, 参考的文档：https://seatunnel.apache.org/docs/2.3.2/start-v2/locally/deployment
2.启动后，编写mysql_to_doris_v2配置文件,配置内容见SeaTunnel Config
3.使用命令提交任务 ./bin/seatunnel.sh --config ./config/mysql_to_doris_v2

### SeaTunnel Version

SeaTunnel 2.3.2

### SeaTunnel Config

```conf
"env" : {
        "execution.parallelism" : 2,
        "job.mode" : "STREAMING"
    },
    "source" : [
        {
            "base-url" : "jdbc:mysql://xxx:15024/test",
            "password" : "xxxxx",
            "parallelism" : 1,
            "table-names" : [
                "test.t_corp_base"
            ],
            "result_table_name" : "fake",
            "plugin_name" : "MySQL-CDC",
            "server-id" : 5656,
            "username" : "xxxx"
        }
    ],
    "sink" : [
        {
            "password" : "xxxxxxx",
            "fenodes" : "xxxxx:8030",
            "sink.enable-2pc" : "true",
            "doris.config" : {
                "format" : "json",
                "read_json_by_line" : "true"
            },
            "table.identifier" : "test.t_corp_base",
            "plugin_name" : "Doris",
            "sink.label-prefix" : "test_json",
            "username" : "root"
        }
    ]
}
```

### Running Command

```shell
启动：./bin/seatunnel-cluster.sh -d
提交任务：./bin/seatunnel.sh --config ./config/mysql_to_doris_v2
```

### Error Exception

```log
2023-07-18 14:13:20,677 INFO  org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand - Closed metrics executor service ......
2023-07-18 14:13:20,677 ERROR org.apache.seatunnel.core.starter.SeaTunnel - 

===============================================================================


2023-07-18 14:13:20,677 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Fatal Error, 

2023-07-18 14:13:20,677 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Please submit bug report in https://github.com/apache/seatunnel/issues

2023-07-18 14:13:20,677 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Reason:SeaTunnel job executed failed 

2023-07-18 14:13:20,678 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Exception StackTrace:org.apache.seatunnel.core.starter.exception.CommandExecuteException: SeaTunnel job executed failed
        at org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand.execute(ClientExecuteCommand.java:188)
        at org.apache.seatunnel.core.starter.SeaTunnel.run(SeaTunnel.java:40)
        at org.apache.seatunnel.core.starter.seatunnel.SeaTunnelClient.main(SeaTunnelClient.java:34)
Caused by: org.apache.seatunnel.engine.common.exception.SeaTunnelEngineException: org.apache.seatunnel.engine.server.checkpoint.CheckpointException: CheckpointCoordinator inside have error.
        at org.apache.seatunnel.engine.server.checkpoint.CheckpointCoordinator.handleCoordinatorError(CheckpointCoordinator.java:253)
        at org.apache.seatunnel.engine.server.checkpoint.CheckpointCoordinator.reportCheckpointErrorFromTask(CheckpointCoordinator.java:319)
        at org.apache.seatunnel.engine.server.checkpoint.CheckpointManager.reportCheckpointErrorFromTask(CheckpointManager.java:182)
        at org.apache.seatunnel.engine.server.checkpoint.operation.CheckpointErrorReportOperation.run(CheckpointErrorReportOperation.java:44)
        at com.hazelcast.spi.impl.operationservice.Operation.call(Operation.java:189)
        at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.call(OperationRunnerImpl.java:273)
        at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:248)
        at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:213)
        at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:175)
        at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:139)
        at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.executeRun(OperationThread.java:123)
        at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:102)
Caused by: org.apache.seatunnel.common.utils.SeaTunnelException: org.apache.seatunnel.common.utils.SeaTunnelException: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://xxxx:15024/test
        at org.apache.seatunnel.connectors.cdc.base.dialect.JdbcDataSourceDialect.openJdbcConnection(JdbcDataSourceDialect.java:54)
        at org.apache.seatunnel.connectors.seatunnel.cdc.mysql.source.MySqlDialect.discoverDataCollections(MySqlDialect.java:83)
        at org.apache.seatunnel.connectors.seatunnel.cdc.mysql.source.MySqlDialect.discoverDataCollections(MySqlDialect.java:46)
        at org.apache.seatunnel.connectors.cdc.base.source.IncrementalSource.createEnumerator(IncrementalSource.java:194)
        at org.apache.seatunnel.engine.server.task.SourceSplitEnumeratorTask.restoreState(SourceSplitEnumeratorTask.java:183)
        at org.apache.seatunnel.engine.server.checkpoint.operation.NotifyTaskRestoreOperation.lambda$null$0(NotifyTaskRestoreOperation.java:106)
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1640)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://xxxx:15024/test
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114)
        at com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)
        at com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114)
        at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108)
        at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPoolFactory.createPooledDataSource(JdbcConnectionPoolFactory.java:52)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPools.getOrCreateConnectionPool(JdbcConnectionPools.java:56)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionFactory.connect(JdbcConnectionFactory.java:60)
        at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:890)
        at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:885)
        at io.debezium.jdbc.JdbcConnection.connect(JdbcConnection.java:418)
        at org.apache.seatunnel.connectors.cdc.base.dialect.JdbcDataSourceDialect.openJdbcConnection(JdbcDataSourceDialect.java:52)
        ... 9 more
Caused by: java.sql.SQLException: No suitable driver
        at java.sql.DriverManager.getDriver(DriverManager.java:315)
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106)
        ... 20 more

        ... 11 more

        at org.apache.seatunnel.engine.client.job.ClientJobProxy.waitForJobComplete(ClientJobProxy.java:122)
        at org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand.execute(ClientExecuteCommand.java:181)
        ... 2 more
 
2023-07-18 14:13:20,678 ERROR org.apache.seatunnel.core.starter.SeaTunnel - 
===============================================================================



Exception in thread "main" org.apache.seatunnel.core.starter.exception.CommandExecuteException: SeaTunnel job executed failed
        at org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand.execute(ClientExecuteCommand.java:188)
        at org.apache.seatunnel.core.starter.SeaTunnel.run(SeaTunnel.java:40)
        at org.apache.seatunnel.core.starter.seatunnel.SeaTunnelClient.main(SeaTunnelClient.java:34)
Caused by: org.apache.seatunnel.engine.common.exception.SeaTunnelEngineException: org.apache.seatunnel.engine.server.checkpoint.CheckpointException: CheckpointCoordinator inside have error.
        at org.apache.seatunnel.engine.server.checkpoint.CheckpointCoordinator.handleCoordinatorError(CheckpointCoordinator.java:253)
        at org.apache.seatunnel.engine.server.checkpoint.CheckpointCoordinator.reportCheckpointErrorFromTask(CheckpointCoordinator.java:319)
        at org.apache.seatunnel.engine.server.checkpoint.CheckpointManager.reportCheckpointErrorFromTask(CheckpointManager.java:182)
        at org.apache.seatunnel.engine.server.checkpoint.operation.CheckpointErrorReportOperation.run(CheckpointErrorReportOperation.java:44)
        at com.hazelcast.spi.impl.operationservice.Operation.call(Operation.java:189)
        at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.call(OperationRunnerImpl.java:273)
        at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:248)
        at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:213)
        at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:175)
        at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:139)
        at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.executeRun(OperationThread.java:123)
        at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:102)
Caused by: org.apache.seatunnel.common.utils.SeaTunnelException: org.apache.seatunnel.common.utils.SeaTunnelException: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://xxxx:15024/test
        at org.apache.seatunnel.connectors.cdc.base.dialect.JdbcDataSourceDialect.openJdbcConnection(JdbcDataSourceDialect.java:54)
        at org.apache.seatunnel.connectors.seatunnel.cdc.mysql.source.MySqlDialect.discoverDataCollections(MySqlDialect.java:83)
        at org.apache.seatunnel.connectors.seatunnel.cdc.mysql.source.MySqlDialect.discoverDataCollections(MySqlDialect.java:46)
        at org.apache.seatunnel.connectors.cdc.base.source.IncrementalSource.createEnumerator(IncrementalSource.java:194)
        at org.apache.seatunnel.engine.server.task.SourceSplitEnumeratorTask.restoreState(SourceSplitEnumeratorTask.java:183)
        at org.apache.seatunnel.engine.server.checkpoint.operation.NotifyTaskRestoreOperation.lambda$null$0(NotifyTaskRestoreOperation.java:106)
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1640)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://xxxx:15024/test
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114)
        at com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)
        at com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114)
        at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108)
        at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPoolFactory.createPooledDataSource(JdbcConnectionPoolFactory.java:52)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPools.getOrCreateConnectionPool(JdbcConnectionPools.java:56)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionFactory.connect(JdbcConnectionFactory.java:60)
        at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:890)
        at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:885)
        at io.debezium.jdbc.JdbcConnection.connect(JdbcConnection.java:418)
        at org.apache.seatunnel.connectors.cdc.base.dialect.JdbcDataSourceDialect.openJdbcConnection(JdbcDataSourceDialect.java:52)
        ... 9 more
Caused by: java.sql.SQLException: No suitable driver
        at java.sql.DriverManager.getDriver(DriverManager.java:315)
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106)
        ... 20 more

        ... 11 more

        at org.apache.seatunnel.engine.client.job.ClientJobProxy.waitForJobComplete(ClientJobProxy.java:122)
        at org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand.execute(ClientExecuteCommand.java:181)
        ... 2 more
2023-07-18 14:13:20,680 INFO  org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand - run shutdown hook because get close signal
```

### Flink or Spark Version

无

### Java or Scala Version

openjdk version "1.8.0_302"
OpenJDK Runtime Environment (build 1.8.0_302-b08)
OpenJDK 64-Bit Server VM (build 25.302-b08, mixed mode)

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by luojinjiang2** [comment]
有下载Mysql的驱动，./lib包下有mysql-connector-java-8.0.30.jar
/connectors/seatunnel目录下有这些连接器：
connector-assert-2.3.2.jar     connector-console-2.3.2.jar  connector-fake-2.3.2.jar     connector-slack-2.3.2.jar      connector-tablestore-2.3.2.jar
connector-cassandra-2.3.2.jar  connector-doris-2.3.2.jar    connector-iceberg-2.3.2.jar  connector-socket-2.3.2.jar
connector-cdc-mysql-2.3.2.jar  connector-email-2.3.2.jar    connector-jdbc-2.3.2.jar     connector-starrocks-2.3.2.jar

**Comment by Xuzhengz** [comment]
你要从官网下载 dev分支的代码 然后下载依赖，自己重新打个包 就行了 ，mysql 驱动8.0.27就行，我之前跟你一样。

**Comment by luojinjiang2** [comment]
> 你要从官网下载 dev分支的代码 然后下载依赖，自己重新打个包 就行了 ，mysql 驱动8.0.27就行，我之前跟你一样。

对,用dev分支打包确实可以了

**Comment by luojinjiang2** [comment]
> > 你要从官网下载 dev分支的代码 然后下载依赖，自己重新打个包 就行了 ，mysql 驱动8.0.27就行，我之前跟你一样。
> 
> 对,用dev分支打包确实可以了

如遇到有些依赖无法下载问题,可以参考该文档:https://kpretty.tech/archives/seatunnel-contributors

**Comment by hailin0** [comment]
Root cause： Environment variable '$SEATUNNEL_HOME' points to the old version dir

**Comment by dzygoon** [comment]
> Root cause： Environment variable '$SEATUNNEL_HOME' points to the old version dir

Was this issue fixed in version 2.3.3?
